homeassistant/components/mill/const.py

```diff
@@ -2,13 +2,16 @@
 
 ATTR_AWAY_TEMP = "away_temp"
 ATTR_COMFORT_TEMP = "comfort_temp"
+ATTR_MAX_HEATING_POWER = "max_heating_power"
 ATTR_ROOM_NAME = "room_name"
 ATTR_SLEEP_TEMP = "sleep_temp"
 BATTERY = "battery"
 CLOUD = "Cloud"
 CONNECTION_TYPE = "connection_type"
 CONSUMPTION_TODAY = "day_consumption"
 CONSUMPTION_YEAR = "year_consumption"
+CONTROL_SIGNAL = "control_signal"
+CURRENT_POWER = "current_power"
```

Comment:
These two constants are single-use.
I know it aligns with the other sensors, but I would suggest to just use a string in the `SensorEntityDescription` for the new sensors.